pkg/replicate/replicator.go

```diff
@@ -160,8 +164,24 @@ func RunReplicate(
 	}, []string{"result"})
 	replicationRunDuration.WithLabelValues(labelSuccess)
 	replicationRunDuration.WithLabelValues(labelError)
+	blocksCleaned := promauto.With(reg).NewCounter(prometheus.CounterOpts{
```

Comment:
Let's remove the cleaning part from this PR as we discussed. It's not needed, part of replicator functionality I believe.